### ISSUE TITLE
Option to exclude frames on hole edge from Overview

### ISSUE THREAD

**Issue by Matt-Power** [Bruker-Amics/bugs_features#388]
v3.3.0.1701

Overview surveys image a lot of the surrounding holder - can we have an option to exclude fields that touch the holder please? This would save a lot of time.

Normally I would just use a BSE only measurement to do this but the binary image detection kicks in making it impossible.

**Comment by jonathan-knapp** [comment]
Good idea. Won't be in this release unless you think this is very important.

**Comment by MarkGuo7** [comment]
@Matt-Power Hi Matt

That's a really good suggestion. Could you send me couple of survey image with sample holders?
There will be a image file locate in your result folder named SampleHolderImages. If you can send me this file and the related ivst file, that would be great.

![image.png](https://images.zenhubusercontent.com/60a59e61f0254ad3e154481a/6848879a-164d-4881-ab18-16c775a12390)

In my case the ivst file should be in the parent folder named DualImageTest.ivst

Thanks
